### PR TITLE
Revise module 2

### PR DESCRIPTION
**Pull request by joelostblom**
- Add intro sentence
- Clarify intro and reference the main resource
- Split into to slides for increased clarity

### CODE REVIEW

**Comment by joelostblom** [comment]
@flor14  This is ready for review! I waited with adding the new "practice" emoji unless we decide which one to use in the other PR, but everything else is here

File: chapters/en/slides/module2/module2_01.md [hunk 1]
```diff
@@ -39,36 +44,37 @@ Essentially,
...
 Luckily,
 we are not doing this anymore;
-instead we have graphical user interface with menus and buttons,
-which is what you are commonly using on your laptop.
+instead we have graphical user interfaces with menus and buttons,
+which is what you are commonly using when interacting with a computer 
+that runs an operating system such as MacOS, Linux, or Windows.
 
-These graphical interface can be thought of as a layer (or *shell*)
+These graphical interface can be thought of as a layer (or a *shell*)
```
Comment (flor14) [inline chapters/en/slides/module2/module2_01.md:55]:
```suggestion
These graphical interface can be thought of as a layer (or a [shell](https://glosario.carpentries.org/en/#shell))
```

File: chapters/en/slides/module2/module2_03.md [hunk 2]
```diff
@@ -6,7 +6,7 @@ type: slides
 # Moving Around
 
 Notes: In this module,
```
Comment (flor14) [inline chapters/en/slides/module2/module2_03.md:8]:
```suggestion
Notes: In this module,
```

Is it necessary to say again "In this module" ?

**Review by flor14 — COMMENTED** [review]
As this is based on a book chapter i have not problems with the text, but I will pay attention the connection between the LOs and the exercises.
- I could be wrong but there are parts of the "Intermediate shell topics"  that doesn't seem to fit in any of the LO.
- In the exercises you uses the concept of parent directory, I don't find it very explain on the slides. Could be?
-  In my opinion if this is a LO: 
image
there should be at least an exercise about it What do you think? 😃

**Review by joelostblom — COMMENTED** [review]
@flor14 Thanks for you comments! I have made most of the specific changes or indicated with a reply to your comment if I haven't. Regarding the three bigger changes you mentioned:

- Good point, I added an LO about pipes as well.
- The parent directory is explained slide deck 3, slide number 6, let me know if you think there is something specific missing and I can add it
- I elaborated on the manual and added an MCW exercise, good catch! 

I am going to squash and merge this, but feel free to respond to the comments as you see fit and if if there is anything else we need to add we can do so in a follow up PR.

**Pull request merged**